smithy-openapi/src/main/java/software/amazon/smithy/openapi/fromsmithy/protocols/AbstractRestProtocol.java

```diff
@@ -111,6 +117,25 @@ abstract Schema createDocumentSchema(
             MessageType messageType
     );
 
+
+    /**
+     * Converts Smithy values in Node form to a data exchange format used by a protocol (e.g., XML).
+     * Then returns the converted value as a long string (escaping where necessary).
+     * If data exchange format is JSON (e.g., as in restJson1 protocol),
+     * method should return values without any modification.
+     *
+     * @param values    values to be converted.
+     * @return  the long string (escaped where necessary) of values in a data exchange format used by a protocol.
+     *
+     * Used for the value property of OpenAPI example objects.
```

Comment:
Docs go before `@param` and `@return`. Separate the second paragraph from the first using  a line that contains `<p>`

Reply:
resolved.